src/Drivers/IpApi.php

```diff
@@ -13,7 +13,7 @@ class IpApi extends Driver
      */
     protected function url($ip)
     {
-        return "http://ip-api.com/json/$ip";
+        return "http://ip-api.com/json/$ip?fields=status,message,country,countryCode,region,regionName,city,zip,lat,lon,timezone,currency,isp,org,as,query";
```

Comment:
The `fields` parameter doesn't appear to be necessary to return the `currencyCode`.

Reply:
So the `IpApi Driver` by default does not return the `currency code` unless specified in the fields status. Hence to return the `currence code` I have to specify in the fields status together with other default response.

Reply:
Ah gotcha, you're right @chandachewe10, my mistake! Thanks for clarifying this 🙏